src/Content/Item.php

```diff
@@ -127,16 +127,6 @@ public static function replaceTag(&$body, &$inform, $profile_uid, $tag, $network
 			$tag_type = substr($tag, 0, 1);
 			//is it already replaced?
 			if (strpos($tag, '[url=')) {
-				// Checking for the alias that is used for OStatus
```

Comment:
Why is this removed?

Reply:
Assigned on line 136, `$newtag` isn't used because of the early `return` statement on line 140. As a result, the `$data` assignment on line 133 also is superfluous and the `preg_match` on line 132 is as well, and finally the comment doesn't seem to reference anything anymore.